docker_for_ctp/make_for_container.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+dir=`pwd`
+VERSION=1.0
+IMAGE_NAME=ctp_container
```

Comment:
IMO it better has prefix such as `cubrid` to distinguish between this project and other containers, for instance,

```sh
PREFIX=cubrid
IMAGE_NAME="${PREFIX}_ctp_container"
```

`CONTAINER_NAME` as well.

Reply:
It's up to you @swi0110 whether you take mind feedback or not. feel free :-)

Reply:
Thanks for your opinion, I will reflect on it.